services/scripts.py

```diff
@@ -1,15 +1,49 @@
 # Music recommendation application
 import csv
-from .cfg import ARTIST_INDEX, LIKES_INDEX
+
+from cfg import TableEnum
+
+
+class Song:
+    def __init__(self, row: list[str]) -> None:
+        self.id = int(row[TableEnum.ID.value])
+        self.name = row[TableEnum.Track.value]
+        self.album = row[TableEnum.Album.value]
+        self.artist = row[TableEnum.Artist.value]
+
+
+class DataBase:
+    def __init__(self, filename):
+        self.db = {}
+
+        with open(filename, encoding='utf-8-sig') as file:
+            _ = file.readline()
+            reader = csv.reader(file)
+
+            for row in reader:
+                self.add_song(Song(row))
+
+    def add_song(self, song: Song) -> None:
+        self.db[song.id] = song
+
+    def get_song(self, song_id: int) -> Song:
```

Comment:
get method will return None if there is not song with this id. So type hint must be not "Song" but "Song | None"

Reply:
i think it should be fixed, probably not with `get`, but it would be better, if we check this case in the future